discovery/aws_service_discovery_instance.go

```diff
@@ -125,6 +126,7 @@ func (a *awsInstance) updateTimeout(timeoutSeconds int64) error {
 
 func (a *awsInstance) start() {
 	go func() {
+		log.Debugf("[Service Discovery](AWS) - Discovery job starting")
```

Comment:
We could use the `logrus` Fields rather than formatting the message this way.

Something as:

```go
log := logrus.New().WithFields(logrus.Fields{"Service Discovery": "AWS"})

log.Debugf("Discovery job starting for ID %s", *a.params.ID)
```